sentry-android-core/src/main/java/io/sentry/android/core/SentryPerformanceProvider.java

```diff
@@ -1,6 +1,6 @@
 package io.sentry.android.core;
 
-import static io.sentry.Sentry.STARTUP_PROFILING_CONFIG_FILE_NAME;
+import static io.sentry.Sentry.App_START_PROFILING_CONFIG_FILE_NAME;
```

Comment:
```suggestion
import static io.sentry.Sentry.APP_START_PROFILING_CONFIG_FILE_NAME;
```
I guess find+replace got you here :P

Reply:
yeah, i started using case sensitive replace after a while, too, but it was too late 😅